lib/Doctrine/DBAL/Driver/PDOSqlsrv/Connection.php

```diff
@@ -34,7 +34,7 @@ class Connection extends PDOConnection implements \Doctrine\DBAL\Driver\Connecti
     public function __construct($dsn, $user = null, $password = null, array $options = null)
     {
         parent::__construct($dsn, $user, $password, $options);
-        $this->setAttribute(\PDO::ATTR_STATEMENT_CLASS, array(Statement::class, array()));
+        $this->setAttribute(\PDO::ATTR_STATEMENT_CLASS, array('Doctrine\\DBAL\\Driver\\PDOSqlsrv\\Statement', array()));
```

Comment:
Escaping backslashes is redundant in single quotes.

Reply:
True. Clearly, I wrote the patch in too much of a hurry. :)